pkg/registry/resource/resourceclaimtemplate/storage/storage.go

```diff
@@ -33,16 +36,17 @@ type REST struct {
 }
 
 // NewREST returns a RESTStorage object that will work against ResourceClass.
-func NewREST(optsGetter generic.RESTOptionsGetter) (*REST, error) {
+func NewREST(optsGetter generic.RESTOptionsGetter, nsClient v1.NamespaceInterface) (*REST, error) {
```

Comment:
if nsClient is nil, return an error (it'll panic later if an adminAccess object is attempted to be created, we want to detect/error at construction time rather than later at runtime)